include/libopencm3/stm32/common/i2c_common_v1.h

```diff
@@ -365,6 +371,23 @@ specific memorymap.h header before including this header file.*/
  * TRISE[5:0]: Maximum rise time in Fast/Standard mode (master mode)
  */
 
+/* --- I2Cx_FLTR values ----------------------------------------------------- */
+
+/*
+ * Bits [3:0] DNF - Digital noise filter
+ * These bits configure a digital noise filter on SDA and SCL pins.
+ * Value sets minimum pulse width needed to trigger i2c operations.
+ * 0 disables, 1 - 15 set minimum width to 'n' * TPCLK1
+ */
+#define	I2C_FLTR_DNF_MASK	0xf
+#define I2C_FLTR_DNF_SHIFT	0
```

Comment:
do you have a whitespace issue here?

Reply:
Yeah, I tend to use 4 space tabs (because screen width) I'll fix these up.